website/src/components/ExecutionQueue/ExeQueue.jsx

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2023 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+
+
+const ExeQueue = ({data}) => {
+    return (
+        <tr>
+            <td><a target='_blank' href={`https://github.com/vitessio/vitess/commit/${data.git_ref}`}>{data.git_ref.slice(0, 8)}</a></td>
+            <td>{data.source}</td>
+            <td>{data.type_of.slice(0, 4)}</td>
```

Comment:
We should display the full `type_of` field. Some benchmark types are longer than four chars, for instance: `oltp-readonly-olap`.